--- Dockerfile.j2	2017-01-19 13:54:07.533262012 +0100
+++ Dockerfile.j2.mitaka	2017-01-19 14:08:08.960824525 +0100
@@ -9,7 +9,6 @@
 
 {{ copy_sources("openstack/horizon", "/horizon") }}
 {{ copy_sources("openstack/sahara-dashboard", "/sahara-dashboard") }}
-{{ copy_sources("openstack/murano-dashboard", "/murano-dashboard") }}
 {{ copy_sources("openstack/searchlight-ui", "/searchlight-ui") }}
 
 RUN mkdir -p /etc/openstack-dashboard /home/horizon /var/www/cgi-bin/horizon \
@@ -19,13 +18,9 @@
         cp -a /searchlight-ui/searchlight_ui/conf/searchlight_policy.json /horizon/openstack_dashboard/conf/; fi) \
     && (if [ -f /searchlight-ui/searchlight_ui/local_settings.d/_1001_search_settings.py ]; then \
         cp -a /searchlight-ui/searchlight_ui/local_settings.d/_1001_search_settings.py /horizon/openstack_dashboard/local/local_settings.d/; fi) \
-    && /var/lib/microservices/venv/bin/pip install --upgrade /horizon /sahara-dashboard /murano-dashboard /searchlight-ui \
     && useradd --user-group horizon \
     && ln -s /etc/openstack-dashboard/local_settings /var/lib/microservices/venv/lib/python2.7/site-packages/openstack_dashboard/local/local_settings.py \
     && cp -r /horizon/openstack_dashboard/conf/* /etc/openstack-dashboard/ \
-    && cp -r /murano-dashboard/muranodashboard/conf/* /var/lib/microservices/venv/lib/python2.7/site-packages/openstack_dashboard/conf/ \
-    && cp -a /murano-dashboard/muranodashboard/local/enabled/* /var/lib/microservices/venv/lib/python2.7/site-packages/openstack_dashboard/local/enabled/ \
-    && cp -a /murano-dashboard/muranodashboard/local/local_settings.d/* /var/lib/microservices/venv/lib/python2.7/site-packages/openstack_dashboard/local/local_settings.d/ \
     && cp /horizon/manage.py /var/lib/microservices/venv/bin/manage.py \
     && /var/lib/microservices/venv/bin/python /var/lib/microservices/venv/bin/manage.py collectstatic --noinput --clear \
     && chown -R horizon: /etc/openstack-dashboard /home/horizon /var/lib/microservices/venv/lib/python2.7/site-packages/static /horizon \
@@ -36,9 +31,6 @@
 # Workaround for https://bugs.launchpad.net/osprofiler/+bug/1361235
 RUN sed -i "s#'../..'#os.path.realpath('../..')#" /var/lib/microservices/venv/lib/python2.7/site-packages/openstack_dashboard/wsgi/django.wsgi
 
-# TODO (amogylchenko): remove ugly sed for session_engine
-RUN sed -i "s/^SESSION_ENGINE = .*/SESSION_ENGINE = 'django.contrib.sessions.backends.cache'/g" /var/lib/microservices/venv/lib/python2.7/site-packages/openstack_dashboard/local/local_settings.d/_50_murano.py
-
 COPY daemon.sh /usr/local/bin/daemon.sh
 COPY horizon_sudoers /etc/sudoers.d/horizon_sudoers
 RUN chmod 755 /usr/local/bin/daemon.sh \
